Documentation/project-docs/platform-names.md

```diff
@@ -0,0 +1,61 @@
+
+That's why referring to .NET Core as 'Store' is no longer correct. But you can think of today's .NET Core as an evolution of the original surface area we used for store applications. Many of the original design goals are still relevant, especially around layering and portability.
+
+## Universal Windows Platform (UWP)
+
+**Also referred to as**: Store, WinRT, Metro
+
+The Universal Windows Platform (UWP) is the platform that is used for building modern, touch-enabled Windows applications as well as headless devices for Internet of Things (IoT). It's designed to unify the different types of devices that you may want to target, including PCs, tablets, phablets, phones, and even the Xbox.
+
+UWP provides many services, such as a centralized app store, an execution environment (AppContainer), and a set of Windows runtime APIs (WinRT). While UWP itself doesn't provide .NET APIs, the Visual Studio tooling experience will present UWP as a platform that is composed of both, WinRT APIs, as well as .NET APIs. The .NET side is provided by .NET Core.
+
+## .NET Native
+
+**Also referred to as**: ahead-of-time (AOT), IL compiler (ILC)
+
+.NET Native is a compiler tool chain that will produce native code ahead-of-time (AOT), as opposed to just-in-time (JIT). The compilation can happen on the developer machine as well as on the store side, which allows blending AOT with the benefits of servicing.
+
+You can think of .NET Native as an evolution of NGEN: NGEN basically simply runs the JIT up front, the code quality and behavior is identical to the JITed version. Another downside of NGEN is that it happens on the user's machine, rather than the developer's machine. NGEN is also at the module level, i.e. for each MSIL assembly there is a corresponding NEGEN'ed assembly that contains the native code. .NET Native on the other hand is a C++ like compiler and linker. It will remove unused code, spend more time optimizing it, and produce a single, merged module that represents the closure of the application.
+
+UWP was the first application model that was supported by .NET Native. We now also support building native console applications for Windows, Mac and Linux.
+
+## Rotor
+
+**Also referred to as**: Shared Source Common Language Infrastructure (SSCLI)
+
+Pretty much at the same time the .NET Framework was released, Microsoft also published Rotor, which is the source code for an implementation of ECMA 335 (Common Language Infrastructure), which is the specification behind .NET.
+
+While parts of the source were identical with the .NET Framework, many pieces had prototypic implementations instead: the purpose of Rotor wasn't to provide a production ready .NET implementation but to provide a platform for research, academia, and validation that the ECMA 335 specification itself can be implemented.
+
+It's also worth pointing out that the source code of Rotor was not an open source license (i.e. not approved by OSI).
```

Comment:
I don't think this sentence means what it's supposed to mean. Perhaps "was not released under an open source license", as it is obvious that the source code itself is not a license.